torch/csrc/distributed/rpc/message.cpp

```diff
@@ -68,10 +68,25 @@ void Message::setId(int64_t id) {
 
 std::vector<c10::weak_intrusive_ptr<c10::StorageImpl>> Message::getStorages()
     const {
+  // Sparse tensors do not have storage. Instead, a sparse tensor
+  // contains two tensors indices and values, and both contain storage.
   std::vector<c10::weak_intrusive_ptr<c10::StorageImpl>> storages;
-  storages.reserve(tensors_.size());
-  for (const auto& tensor : tensors_) {
-    storages.emplace_back(tensor.storage().getWeakStorageImpl());
+  int64_t storage_pointers = 0;
```

Comment:
nit: var and method names in this file are using camel instead of snake convention.

Reply:
Okay, that was a mistake.